cockroachcloud/create-your-cluster.md

```diff
@@ -103,27 +113,31 @@ The choice of hardware per node determines the [cost](#step-2-select-the-cloud-p
     Buffer | Additional buffer (overhead data, accounting for data growth, etc.). If you are importing an existing dataset, we recommend you provision at least 50% additional storage to account for the import functionality.
     Compression | The percentage of savings you can expect to achieve with compression. With CockroachDB's default compression algorithm, we typically see about a 40% savings on raw data size.
 
-    For more detailed disk performance numbers, see the relevant [GCP](https://cloud.google.com/compute/docs/disks/performance) and [AWS](https://aws.amazon.com/ebs/features/#Amazon_EBS_volume_types) documentation.
+    For more details about disk performance on a given cloud provider, refer to: <ul><li><b>GCP</b>: <a href="https://cloud.google.com/compute/docs/disks/performance">Configure disks to meet performance requirements</a></li><li><b>AWS</b>: <a href="https://aws.amazon.com/ebs/features/#Amazon_EBS_volume_types">Amazon EBS volume types</a></li><li><b>Azure</b>: <a href="https://learn.microsoft.com/en-us/azure/virtual-machines/disks-enable-performance">Increase IOPS and throughput limits for Azure Premium SSDs and Standard SSD/HDDs</a></li></ul>
```

Comment:
I'm not comfortable with the link titles here, since customers are not allowed to configure IOPS or anything else for disks other than their size.

Reply:
Do you have a suggestion for a better article on microsoft.com? I used the title they gave the article and it seems strange to re-title it.

Reply:
Replaced the Azure link with https://learn.microsoft.com/en-us/azure/security/fundamentals/encryption-atrest.